convert2rhel/unit_tests/redhatrelease_test.py

```diff
@@ -85,6 +96,82 @@ def test_patch_yum_conf_existing_distroverpkg(self):
             self.assertEqual(yum_conf._yum_conf_content.count("#distroverpkg="), 1)
 
 
+@pytest.mark.parametrize(
+    "subprocess_ret, expected",
+    [
+        ("S.5....T.  c /etc/yum.conf", True),
+        ("", False),
+        (".......T.  c /etc/yum.conf", False),
+    ],
+)
+def test_is_modified_yum(monkeypatch, subprocess_ret, expected):
+
+    is_modified_run(
+        monkeypatch=monkeypatch,
+        pkg_type="yum",
+        subprocess_ret=subprocess_ret,
+        expected=expected,
+    )
+
+
+@pytest.mark.parametrize(
+    "subprocess_ret, expected",
+    [
+        ("S.5....T.  c /etc/dnf/dnf.conf", True),
+        (".......T.  c /etc/dnf/dnf.conf", False),
+        (".M.......  g /var/lib/dnf", False),
+    ],
+)
+def test_is_modified_dnf(monkeypatch, subprocess_ret, expected):
+
+    is_modified_run(
+        monkeypatch=monkeypatch,
+        pkg_type="dnf",
+        subprocess_ret=subprocess_ret,
+        expected=expected,
+    )
+
+
+def test_is_modified_unknown(monkeypatch):
+    is_modified_run(monkeypatch, "unknown", "anything", False)
+
+
+# function called by test_is_modified_yum/dnf/unknown
+def is_modified_run(monkeypatch, pkg_type, subprocess_ret, expected):
+    monkeypatch.setattr(pkgmanager, "TYPE", value=pkg_type)
+
+    run_subprocess = mock.Mock(return_value=(subprocess_ret, "anything"))
+    monkeypatch.setattr(utils, "run_subprocess", value=run_subprocess)
+
+    assert YumConf._is_modified() == expected
+
+
+@pytest.mark.parametrize("is_modified, patch_call", [(True, True), (False, False)])
+def test_patch(monkeypatch, is_modified, patch_call):
```

Comment:
Could `is_modified` and `patch_call` be consolidated to just `is_modified`? Doesn't seem like we need two parameters here

Reply:
Yes, of course :) Sorry for that, I feel little bit awkwardly when I see the line

Reply:
Happens to everyone, don't worry about it!